clients/fides-js/src/lib/consent-types.ts

```diff
@@ -782,6 +782,7 @@ export enum ConsentMethod {
   GPC = "gpc",
   INDIVIDUAL_NOTICE = "individual_notice",
   ACKNOWLEDGE = "acknowledge",
+  OT_MIGRATION = "ot_migration",
```

Comment:
This feels a little too specific. If we started migrating a different competitor tomorrow, would we want to add another new method here? That's perfectly valid if we do, but want to call it out to make sure it's intentional that way.

Reply:
This is something I'd asked Michael about, and we agreed on this naming. Our consent reporting today is very basic and does not allow us to see much granularity with various facets like source / method / etc, so I'm OK being more specific rather than the other way around here.